### DISCUSSION TITLE
How to get started (TUTORIAL)

### DISCUSSION THREAD

**Discussion by Mihaug9** [nkaz001/hftbacktest#76] [General]
Please can you show the step by step process of how to apply this repo for newbies like us?

Like is it meant for windows, linux, both?
How does one set it up? What are the steps?

How does one connect their binance futures account with api keys?

How does one adjust parameters (Wallet exposure, order type, trading parameters etc)

I am sure alot of people will appreciate this also if you could do a brief video of how to set it up on a fresh instance that will also be of great help. Thanks

**Comment by nkaz001** [comment]
Thank you for your suggestion.

I assume you are referring to the hftbacktest Rust implementation, which is designed for not just backtesting but a live trading bot. I will provide more detailed documentation once the implementation has matured. Currently, it's in the early stages of development and has not yet been comprehensively tested. However, I am confident it will function on both Windows, Linux, though Linux is the recommended platform.

hftbacktest is primarily a high-frequency-trading backtesting tool that facilitates a smooth transition to live trading; it is not a plug-and-play trading bot. I strongly recommend gaining a thorough understanding of Rust and the codebase, learning how the examples function, and then developing your own version.
